packages/app/utils/modal.ts

```diff
@@ -37,6 +39,32 @@ export const showSendModal = (configs?: SendModalConfig) => {
 	});
 };
 
+interface RequirePasscodeModalConfig {
+	title: string;
+	desc: string;
+	onPasscodeComplete?: (passcode: string) => Promise<{
+		responseCode?: ResponseCode | undefined;
+		message?: string | undefined;
+	}>;
+	onActionComplete?: () => void;
+}
```

Comment:
Is it possible to reuse this interface from `../modals/RequirePasscodeModal` and avoid duplicating code